docs/fundamentals/artifacts.md

```diff
@@ -1,193 +1,77 @@
+pointing directly to the original training task's model.
 
-### List of Supported Artifacts
+## Manually Logging Models
 
-- Numpy array (as npz file)
-- Pandas dataframe
-- PIL (converted to jpg)
-- Files and folders
-- Python objects (pickled)
+### Output Models
 
-## Models 
-Models are a special kind of artifact and, unlike regular artifacts, which can only be accessed with the creating Task's ID,
-Models are entities with their own unique ID that can be accessed directly or via the creating task.
+ClearML stores training results as output models. The `OutputModel` object is instantiated with a task object as an 
+argument (see [`task`](../references/sdk/model_outputmodel.md) parameter), so it's automatically registered as the Task’s 
+output model. Since OutputModel objects are connected to tasks, the models are traceable in experiments.
 
-This property makes Models a standalone entry that can be used as an artifactory interface.
+Output models are read-write so weights can be updated throughout training. Additionally, users can specify a model's 
+network design and label enumeration. Once an output model is registered, it can be used as the input model for another 
+experiment.
 
-### Automatic Model Logging
+The snapshots of manually uploaded models aren't automatically captured, but ClearML provides methods to update them 
+through a `Task` or `OutputModel` object.
 
-When models are saved using certain frameworks (for instance, by calling the `torch.save()` method), ClearML automatically 
-logs the models and all snapshot paths.
+### Input Models  
 
-![image](../img/examples_model_update_model.png)
+ClearML provides flexibility for explicitly connecting input models and experimentation, including:
 
-See automatic model logging examples: 
-* [TF](../guides/frameworks/tensorflow/tensorflow_mnist.md)
-* [PyTorch](../guides/frameworks/pytorch/pytorch_mnist.md)
-* [Keras](../guides/frameworks/keras/keras_tensorboard.md)
-* [Scikit-Learn](../guides/frameworks/scikit-learn/sklearn_joblib_example.md)
-* [XGBoost](../guides/frameworks/xgboost/xgboost_sample.md)
-* [FastAI](../guides/frameworks/fastai/fastai_with_tensorboard.md)
+* Importing pre-trained models from external sources such as Amazon AWS, GIT repositories, PyTorch, and TensorFlow.
+* Using standalone models already registered in ClearML by previously run experiments. 
+* Defining your own input models in scripts
+  
+## Setting Upload Destination
 
+* ClearML automatically captures the storage path of Models created by supported frameworks. By default, it stores the 
+  local path they are saved to.
+* Upload destinations can be specified explicitly on a per OutputModel or per experiment basis. Alternatively, the upload 
+  destination of all OutputModels can be specified in the ClearML [configuration file](../configs/clearml_conf.md). 
 
-### Manual Model Logging 
+## WebApp
```

Comment:
@pollfly Here we're using "WebApp" and in Hyperparameters "WebApp Access". Should be uniform across fundamentals pages (I think "WebApp Interface" make most sense?)

Reply:
switched both to WebApp Interface